app/depict/src/main/java/org/openscience/cdk/depict/SvgDrawVisitor.java

```diff
@@ -198,7 +198,7 @@ String toStr(Color col) {
         if (col.getAlpha() == 255) {
             return String.format("#%06X", (0xFFFFFF & col.getRGB()));
         } else {
-            return String.format("rgba(%d,%d,%d,%.2f)", col.getRed(), col.getGreen(), col.getBlue(), col.getAlpha()/255d);
+            return String.format(Locale.US, "rgba(%d,%d,%d,%.2f)", col.getRed(), col.getGreen(), col.getBlue(), col.getAlpha()/255d);
```

Comment:
I think ``Locale.ROOT`` is the normal way to do this.

**Locale.ROOT states that it is the language/country neutral locale for the locale sensitive operations**

I think it's more if someone comes along and tries to internationalise the code they put the commas back in ;-).

Reply:
I'm on it.